src/main/java/wromaciej/moviesrental/movies/controllers/MoviesController.java

```diff
@@ -27,6 +26,11 @@ public MoviesController(MovieService movieService) {
         return movieService.findMovie(movieId);
     }
 
+    @RequestMapping(value="/{movieId}/rental/{days}")
+    public @ResponseBody double findMovieRentalRateForDays(@PathVariable int movieId, @PathVariable int days) {
+        return movieService.findMovie(movieId).getRentalRate()*days;
```

Comment:
you should never place your business logic in a controller. Please create proper method in e.g. CartService and use MovieService there.